src/Web/Sfa.Das.Sas.Infrastructure/ElasticSearch/FrameworkApiRepository.cs

```diff
@@ -38,5 +38,15 @@ public Framework GetFrameworkById(string id)
 
             return _frameworkMapping.MapToFramework(result);
         }
+
+        public long GetFrameworksAmount()
+        {
+            throw new NotImplementedException();
+        }
+
+        public int GetFrameworksExpiringSoon(int daysToExpire)
+        {
+            throw new NotImplementedException();
```

Comment:
Should we pull all the frameworks and filter on expire after. It just abotu 700 anyway and there will not be any more.
In case we cut elastic?

Same for GetFrameworksAmount()..

/c